com.unity.multiplayer.mlapi/Runtime/Profiling/ProfilingDataStore.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace MLAPI.Profiling
+{
+    public class ProfilingDataStore
+    {
+        readonly Dictionary<string, int> m_Dictionary = new Dictionary<string, int>();
+
+        public void Add(string fieldName, int value)
+        {
+            m_Dictionary.Add(fieldName, value);
+        }
+
+        public void Increment(string fieldName, int count = 1)
```

Comment:
Why not just make the type `ProfilerConstants` and avoid the ToString()

Reply:
IT cant be typed. Each transport will have its own types of strings that it wants to propagate up.

Reply:
Oddly enough, we had the same thing in de-string-ifying the channels in MLAPI.  They too have per-transport customization, so we still established a set of enums in the SDK with a 'LAST_ENTRY' enum, but then if your transport wants more than that you just do LAST_ENTRY + 1, etc.  That let us have our type safety / low overhead but also be extensible.  We do this in Transport.cs:8 and then like in TransportTest.cs:42